examples/html/src/index.ts

```diff
@@ -59,4 +59,4 @@ async function init() {
     (document.getElementById('injectedAccount') as HTMLHeadingElement).innerHTML = address
 }
 
-setTimeout(init, 2000)
\ No newline at end of file
+setTimeout(init, 2000)
```

Comment:
Conflict resolution brought this timeout back

Reply:
Oh, i think you where checking the **prev** commit, 

https://github.com/subspace/subspace.js/blob/445cc2b061aa3dc0cffddef76e8a49c08babb2a7/examples/html/src/index.ts#L62